raisimGymTorch/raisimGymTorch/env/envs/rsg_ouzel/impedance_control_module.cpp

```diff
@@ -140,8 +140,14 @@ namespace rw_omav_controllers {
               com_offset_.cross(linear_accel_B_des);
       angular_accel_B_des += inertia_.inverse() * moment_ff;
 
-      (*wrench_command).thrust = mass_ * linear_accel_B_des;
-      (*wrench_command).torque = inertia_ * angular_accel_B_des;
+      //Init filter with first received commands
+      static lowpassWrench commandFilter(linear_accel_B_des, angular_accel_B_des);
+      //Filter with 40.0rad cutoff frequency. Filter runs at 200hz (same sampling rate of the controller)
+      commandFilter.update(linear_accel_B_des, angular_accel_B_des,40.0,(1.0/200.0));
```

Comment:
So the 40rad cutoff frequency is what you found out by testing it on the full stack simulation right?
The controller frequency is a configuration parameter in the training environment. Currently it is set at 100 Hz. So can I simply replace the 200hz or has the controller rate an impact on the cutoff frquency?